db/revision_cache_lru.go

```diff
@@ -135,6 +149,33 @@ func (rc *LRURevisionCache) getFromCache(docID, revID string, loadOnCacheMiss bo
 	return docRev, err
 }
 
+func (rc *LRURevisionCache) LoadInvalidRevFromBackingStore(key IDAndRev, includeBody bool, includeDelta bool) (DocumentRevision, error) {
```

Comment:
Add comment to describe what an invalid rev is (i.e. a newer version of a revision that was previously cached, but needs to be replaced).

Reply:
Added comments how do they look?